src/Namotion.Reflection/XmlDocsFormatting.cs

```diff
@@ -0,0 +1,129 @@
+                { "strong", (sb, e) => AppendSimpleTaggedElement(sb, e, "<strong>", "</strong>") },
+                { "i", (sb, e) => AppendSimpleTaggedElement(sb, e, "<i>", "</i>") }
+            };
+
+        /// <summary>
+        /// Appends the value of <paramref name="element"/> surrounded by the neccessary HTML-tags to maintain its formatting information (if supported).
+        /// </summary>
+        /// <param name="stringBuilder">The current <see cref="StringBuilder"/>.</param>
+        /// <param name="element">The <see cref="XElement"/> to append.</param>
+        /// <returns>The value of <paramref name="stringBuilder"/>.</returns>
+        private static StringBuilder AppendHtmlFormattedElement(StringBuilder stringBuilder, XElement element)
+        {
+            return AppendMapFormattedElement(stringBuilder, element, htmlTagMap);
+        }
+        #endregion
+
+        #region Markdown formatting
+        private static readonly IDictionary<string, Func<StringBuilder, XElement, StringBuilder>> markdownTagMap =
+            new Dictionary<string, Func<StringBuilder, XElement, StringBuilder>>()
+            {
+                { "c", (sb, e) => AppendSimpleTaggedElement(sb, e, "`", "`") },
+                { "b", (sb, e) => AppendSimpleTaggedElement(sb, e, "**", "**") },
+                { "strong", (sb, e) => AppendSimpleTaggedElement(sb, e, "**", "**") },
+                { "i", (sb, e) => AppendSimpleTaggedElement(sb, e, "*", "*") }
+            };
+
+        /// <summary>
+        /// Appends the value of <paramref name="element"/> surrounded by the neccessary Markdown-tags to maintain its formatting information (if supported).
+        /// </summary>
+        /// <param name="stringBuilder">The current <see cref="StringBuilder"/>.</param>
+        /// <param name="element">The <see cref="XElement"/> to append.</param>
+        /// <returns>The value of <paramref name="stringBuilder"/>.</returns>
+        private static StringBuilder AppendMarkdownFormattedElement(StringBuilder stringBuilder, XElement element)
+        {
+            return AppendMapFormattedElement(stringBuilder, element, markdownTagMap);
+        }
+        #endregion
+
+        /// <summary>
+        /// Appends the value of <paramref name="element"/> surrounded by the tags specified in <paramref name="map"/> (if supported).
+        /// </summary>
+        /// <param name="stringBuilder">The current <see cref="StringBuilder"/>.</param>
+        /// <param name="element">The <see cref="XElement"/> to append.</param>
+        /// <param name="map">Map of formatting tags.</param>
+        /// <returns>The value of <paramref name="stringBuilder"/>.</returns>
+        private static StringBuilder AppendMapFormattedElement(
+            StringBuilder stringBuilder,
+            XElement element,
+            IDictionary<string, Func<StringBuilder, XElement, StringBuilder>> map)
```

Comment:
I can be concrete `Dictionary`, should use that as type

Reply:
Changed to `Dictionary<...>`